themes/default/content/docs/intro/pulumi-service/billing-managers.md

```diff
@@ -17,6 +17,10 @@ Team Editions of the Pulumi Service are limited to 1 Billing Manager at a time.
 
 Billing Manager invitations work much the same way member invitations do, but they are managed on the Billing and Usage page.
 
+{{% notes "info" %}}
+Billing Managers like members must be members of the [organization identity provider]({{< relref "/docs/intro/pulumi-service/organizations#organization-identity-providers" >}}) in order to gain access to the Pulumi organization.
```

Comment:
This could maybe use some commas around `like members`, e.g. "Billing Managers, like members, ...". But I have a tendency to overuse commas so feel free to ignore!